examples/events.py

```diff
@@ -1,217 +1,123 @@
 import coc
 import logging
-import asyncio
-import objgraph
-import psutil
-import tracemalloc
-import random
-from examples import creds
 
-tracemalloc.start()
 
-client = coc.login(creds.email, creds.password, client=coc.EventsClient, key_names="windows", cwl_active=False)
+client = coc.login(email="DevSiteEmail", password="DevSitePassword", client=coc.EventsClient)
+logging.basicConfig(level=logging.INFO)
 log = logging.getLogger()
-logging.basicConfig(level=logging.CRITICAL)
 
-c_tags = [
+
+clan_tags = [
     "#20090C9PR",
     "#202GG92Q",
     "#20C8G0RPL",
     "#20CG8UURL",
     "#20L2GVUCQ",
-    "#20PJQQP0G",
-    "#20RP90PLL",
-    "#20VCVGVCG",
-    "#20YRUQRLJ",
-    "#2229Y88R9",
-    "#228VP82GY",
-    "#22GLCR9Q",
-    "#22J8ULLL",
-    "#22JYYGVUC",
-    "#22LR9QY98",
-    "#22PQL2L0R",
-    "#22QCGYV2Q",
-    "#22RGGJVP",
-    "#22RU00UC9",
-    "#22YCYL89",
-    "#22YRJLVU8",
-    "#280V0VYL",
-    "#282L8GLJ",
-    "#28CJ8RRYJ",
-    "#28GCGPVL2",
-    "#28GQ08L29",
-    "#28Q0R2QV9",
-    "#28QJ2LP8Q",
-    "#28QLCPCLG",
-    "#28QRV0Y0G",
-    "#28ULR99VL",
-    "#28V2RC0C9",
-    "#28VPCGV2G",
-    "#28VQUV9J8",
-    "#28YC02QYC",
-    "#290UJUQVQ",
-    "#2922CY2R",
-    "#292GVY28P",
-    "#2982PCG09",
-    "#298CJ9JPC",
-    "#2998PGPY9",
-    "#29998R8L0",
-    "#299C8QR9L",
-    "#299P28QRL",
-    "#29CV8VLL2",
-    "#29G2JU888",
-    "#29G9QJVLR",
-    "#29GC8JQG0",
-    "#29GLJCLQQ",
-    "#29GLLLURV",
-    "#29GQCC29C",
-    "#29GQQ9GVL",
-    "#29GVQV9Q9",
-    "#29GVYYJ8",
-    "#29L00JCVG",
-    "#29L2U9U9J",
-    "#29LPUGR9L",
-    "#29LPVL99P",
-    "#29LUPU9QV",
-    "#29PJYUCV2",
-    "#29RY29LL2",
-    "#29U9UYUG",
-    "#29VC9GR0Y",
-    "#29Y2QVGL",
-    "#29Y2R0CJQ",
-    "#2C8JV0PG",
-    "#2CR80PVR",
-    "#2CVLP0P0",
-    "#2G2LR0",
-    "#2G9C9CVC",
-    "#2GU2Y0JL",
-    "#2JU0P82U",
-    "#2JUJ2G22",
-    "#2LP2PUUP",
-    "#2P0C9LY8Y",
-    "#2P0QL9C9G",
-    "#2P2JR088R",
-    "#2P2LJJPUY",
-    "#2P2Q9PPJJ",
-    "#2P2UVPRVC",
-    "#2P80Q9LR8",
-    "#2P89GGLYY",
-    "#2P9CGUUR8",
-    "#2P9PC0JC9",
-    "#2PCUQJYGJ",
-    "#2PG8R9GU8",
-    "#2PGJUL98L",
-    "#2PGQY2YPV",
-    "#2PGV2GUUJ",
-    "#2PJJPGJ9U",
-    "#2PLGGYGLV",
-    "#2PLLQRQPP",
-    "#2PLR9VPYP",
-    "#2PPR9VUGC",
-    "#2PQ08LCGR",
-    "#2PQY9GQU9",
-    "#2PRPJU8RY",
-    "#2PUGC20UC",
-    "#2PUL8RU82",
-    "#2PUPY9022",
 ]
 
+player_tags = [
+    "#YQ2QYLGJ",
+    "#QYJCVGL",
+    "#2LURLC9V",
+    "#QCQR298V",
+    "#82CVC2V8",
+    "#29U09V8J",
+    "#8GYGL22P",
+]
+
+
+"""Clan Events"""
+
+
+@client.event  # Pro Tip : client event is mandatory then only any event will work so dont leave :)
+@coc.ClanEvents.member_donations(tags=clan_tags)
+async def on_clan_member_donation(old_donation, new_donation):
```

Comment:
```suggestion
async def on_clan_member_donation(old_member, new_member):
```
and the references to this below